plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraTable.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.cassandra;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.testing.sql.SqlExecutor;
+
+import java.security.SecureRandom;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+
+import static com.google.common.base.Preconditions.checkState;
+import static java.lang.Character.MAX_RADIX;
+import static java.lang.Math.abs;
+import static java.lang.Math.min;
+import static java.lang.String.format;
+import static java.lang.String.join;
+import static java.util.stream.Collectors.joining;
+
+public class TestCassandraTable
```

Comment:
Is it possible to make this extend TestTable? What changes would be needed?

Reply:
> Is it possible to make this extend TestTable?

Yes, by inheriting we can reduce duplicate code, which also makes more sense.

> What changes would be needed?

1. `TestTable` needs to provide a protected method that can be called `createTableName`, allowing subclasses to override the modified table name (because Cassandra table names have a **length limit**, so we need this method to crop the table name)
2. `TestTable` needs to change the `SqlExecutor` scope to `protected`

I'll update this PR later and you can re-review it.

Reply:
Updated.